github-site-plugin/src/main/java/com/github/maven/plugins/site/SiteMojo.java

```diff
@@ -433,7 +433,7 @@ public void execute() throws MojoExecutionException {
 
             CommitUser author = new CommitUser();
             author.setName(user.getName());
-            author.setEmail(user.getEmail());
+            author.setEmail(userService.getEmails().get(0));
```

Comment:
Should this use `user.getEmail()` when available and only fall back to `userService.getEmails().get(0)` when the first is null/empty?